### PR TITLE
Build instructions for macOS

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

**Comment by the-accidental** [comment]
Thanks for testing it on MacOS and contributing!

**Comment by brtkwr** [comment]
This is my friend Tom whom I captured this morning. His hair and beard didn't have many unique features to make it into the model unfortunately. How do I improve the result as the face is a bit coarser than I would like?

tom

**Comment by the-accidental** [comment]
Ok, so a few things you can try but this will be a longish post so:

**TLDR** Try [SMVS](https://github.com/flanggut/smvs) with the `-S` option, run it instead of scripts 2 and 3, then edit 4 so it points at the ply SMVS creates and mesh from there.

OK, lets dive in.

First off, that's actually a pretty good effort for a person with a single camera. Normally for scanning a person you'd use a large multi-camera rig round the person [like this](https://i.pinimg.com/originals/17/4b/16/174b16af5d822c7d1c41d81a27070dd3.jpg). Reason is, if you remember 2 of my 3 things the images need to - sharp, no motion. It's impossible to get a person to stay still enough whilst you move around them taking photos. I mean, if you tried a long exposure of as long as it takes you, it'd be blurry - as you can't hold the camera steady. If you think about what's happing under the hood, you're trying to match pixels between photos to estimate depth, but if the target pixel is moving, you introduce error.

That said, a few things to check. Looks like you're short on images, especially underneath, or they didn't match at the SFM stage. So check how many photos you took, and how many camera positions you had - that'll tell you if some didn't match. If you're having problems at that stage, then a few options. More light (more detail to match) or choose an area to shoot with nice textured surroundings - then the SFM step can locate cameras accurately on these (static) features. 

Assuming the above checks out and you have lots of good camera poses, then your issue looks like sparse points, or it all got lost in the meshing getting rid of noise. If it's all going in the mesh or cleaning - you need more photos in the input, there's too much uncertainty. If the points aren't there at all - you need to look at the depth maps. See where you're getting no good matches and there's no data (pink areas). So, go into step 2 script and play with the options there till you get better depth maps. It could be you've not got enough detail to match a lot of pixels, my script defaults to scaling the images to speed processing. Change the s2 to s0 to use full resolution - then go do something else as it'll take some time to run!

Finally, if the images were taken somewhere with a single light source (natural light or one light) then the assumptions used by SMVS hold, so you can try that as per my intro. This uses the shading to estimate shape rather than feature matching (well, simplified that's an explanation!). So it can work well where small detail is lost or can't be matched - this is what I did for Totoro, and if your subject is bearded it could help.

Anyway, that gives you some starting points to look at, but in general I'd say it's not bad, and doing a person with a single camera is challenging!
